206/step2_recursive2.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+反転したリストの末尾につけるためと、最終的にreturnするための反転リストの先頭を同時に返すパターン。
+関数の中で着目呈しているノード(head)を、それ以降の反転されたリストの末尾に接続することで反転を実現する。
+
+https://github.com/goto-untrapped/Arai60/pull/27/files/14646ec0859dd9411e6983bf6c63e6f15a1f9f32#r1641596128
+>何も渡さずに、6番目以降をひっくり返したものを返してもらう。
+*/
+class Solution {
+public:
+  std::tuple<ListNode*, ListNode*> recursiveReverseList(ListNode* head) {
```

Comment:
細かいですが

```suggestion
  std::tuple<ListNode*, ListNode*> reverseListRecursive(ListNode* head) {
```
処理を表す動詞を先頭に持ってくるのが一般的だと思います

Reply:
ありがとうございます、ちょっとそのあたりの徹底ができておらずでした。